test/Elastica/IndexTest.php

```diff
@@ -835,29 +815,6 @@ public function testAnalyzeExplain()
         $this->assertArrayHasKey('custom_analyzer', $data);
     }
 
-    /**
-     * @group unit
-     */
-    public function testThrowExceptionIfNotScalar()
```

Comment:
Not sure who the removal of this is related? Not needed anymore?

Reply:
As we now use PHP typehings, passing something that is not a scalar will throw a PHP error :)